build-tools/xabootstrap/xabootstrap/ConfigAndData/Dependencies/Windows.cs

```diff
@@ -0,0 +1,6 @@
+namespace Xamarin.Android.Bootstrap.Dependencies
+{
+	partial class Windows : Dependencies
+	{
+	};
```

Comment:
This isn't C++; trailing `;` isn't needed. :-)

Reply:
muscle memory :)